engine/lite/kafka/runtime/src/universal/bin/run.sh

```diff
@@ -33,6 +37,10 @@ if [[ "${RUN_IN_BACKGROUND}" == "true" ]]; then
   echo $! > $PID_FILE
   echo "Nussknacker Kafka Runtime up and running"
 else
+  echo "JVM:"
+  java -version
```

Comment:
maybe it's time we support JAVA_HOME? 
like in: https://github.com/apache/flink/blob/master/flink-dist/src/main/flink-bin/bin/config.sh#L214?

Reply:
I'd rather don't want to spend more time on this issues - let's wait with this until it will be needed